iceoryx2-cal/tests/static_storage_trait_tests.rs

```diff
@@ -260,6 +265,48 @@ mod static_storage {
         assert_that!(<Sut as NamedConceptMgmt>::list().unwrap(), len 0);
     }
 
+    #[test]
+    fn concurrent_create_same_locked_storage_multiple_times_fails_for_all_but_one<
+        Sut: StaticStorage,
+    >() {
+        let _watch_dog = Watchdog::new_with_timeout(Duration::from_secs(5));
```

Comment:
Can you use just `Watchdog::new()`? It comes with a default timeout, and if it is too generous (60s), we can correct it globally.
And I think it is not a problem to be a bit more generous since the watchdog only activates when the test is deadlock that is hopefully a rare cornercase